apps/web/pages/apps/installed.tsx

```diff
@@ -81,122 +81,92 @@ function ConnectOrDisconnectIntegrationButton(props: {
   );
 }
 
-function IntegrationsContainer() {
+type AppOutput = inferQueryOutput<"viewer.integrations">["items"][0];
+
+interface IntegrationsContainerProps {
+  variant: App["variant"];
+  className?: string;
+}
+
+const IntegrationsContainer = ({ variant, className = "" }: IntegrationsContainerProps): JSX.Element => {
   const { t } = useLocale();
-  const query = trpc.useQuery(["viewer.integrations"], { suspense: true });
+  const query = trpc.useQuery(["viewer.integrations", { variant }], { suspense: true });
```

Comment:
As mentioned, we only get the integrations per variant

Reply:
Can we add an input to the query it only returns installed apps? Instead of filtering on the frontend, let's just reduce the data returned from the server.

Reply:
Makes sense!